common/scala/src/main/scala/whisk/core/entity/Limits.scala

```diff
@@ -47,7 +47,7 @@ protected[entity] abstract class Limits {
  * @param timeout the duration in milliseconds, assured to be non-null because it is a value
  * @param memory the memory limit in megabytes, assured to be non-null because it is a value
  */
-protected[core] case class ActionLimits protected[core] (timeout: TimeLimit, memory: MemoryLimit) extends Limits {
+protected[core] case class ActionLimits protected[core] (timeout: TimeLimit, memory: MemoryLimit, logs: Option[LogLimit] = Some(LogLimit())) extends Limits {
```

Comment:
update javadoc.
add comment that log limit is an option for backward compatibility/schema migration.